seeds/002-students.js

```diff
@@ -0,0 +1,12 @@
+
+exports.seed = function(knex) {
+  return knex('students')
+    .truncate() 
+    .then(() => {
+      return knex('students').insert([
+        {name: 'Hack the Gibson'},
+        {name: 'Nick the NSA'},
```

Comment:
these will be successfully entered since they're seeded, but not that you didn't add a cohort_id for them